marimo/_plugins/ui/_impl/input.py

```diff
@@ -232,6 +232,12 @@ def __init__(
             )
         # If steps are provided
         if steps is not None:
+            # Cast to a list in case user passes a numpy array
+            if not isinstance(steps, list):
+                import numpy as np
```

Comment:
we should check if numpy is imported:
`if DependencyManager.numpy.imported():`, otherwise this could error out if they dont have numpy installed

Reply:
Ah! I saw that in the tests but didn't realise it was generally available. Will adapt.